.github/workflows/build-depends.yml

```diff
@@ -51,23 +51,38 @@ jobs:
             depends/sources
           key: depends-sources-${{ hashFiles('depends/packages/*') }}
           restore-keys: |
-            depends-sources-${{ hashFiles('depends/packages/*') }}
             depends-sources-
 
+      - name: Cache SDKs
+        uses: actions/cache@v4
+        if: inputs.build-target == 'mac'
+        with:
+          path: |
+            depends/sdk-sources
+            depends/SDKs
+          key: depends-sdks-${{ hashFiles('depends/hosts/darwin.mk') }}
```

Comment:
can we use https://github.com/dashpay/dash/blob/632b8b43dddf8f0c34c5a0c379c54829422727f2/ci/test/00_setup_env_mac.sh#L13 instead? I think the way you have it now is plausible to have stale cache

Reply:
`depends/hosts/darwin.mk` is what decides the Xcode version, LLD version and uses them to define the SDK path ([source](https://github.com/dashpay/dash/blob/bb469687d3d936f82fd8e8fbe0934eec5e17df5e/depends/hosts/darwin.mk#L1-L8)) used by the compiler ([source](https://github.com/dashpay/dash/blob/bb469687d3d936f82fd8e8fbe0934eec5e17df5e/depends/hosts/darwin.mk#L57)), any bump in the SDK version requires changes to `hosts/darwin.mk`.

Would instead propose to drop those variables from `00_setup_env_mac.sh`.